AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -294,7 +294,7 @@ infranken.de#$#div[id^="cmpbox"] { display: none !important; }
 skillbox.ru##app-cookies
 ||bratpfannentest.de/wp-content/plugins/borlabs-cookie^
 nordbayern.de##div[id^="cmpbox"]
-drweb.com###cookie_accept_msg_block
+free.drweb.ru,drweb.com###cookie_accept_msg_block
```

Comment:
This is necessary for `drweb.ru`.

Reply:
@zloyden You mean false positive? BTW the same cookie consent is present on all locales e.g. `free.drweb.co.jp`.

Reply:
And even wildcard is not enough, German version is `drweb-av.de`

Reply:
I meant that we don't need to use subdomain `free.` because it's needed for `drweb.ru`.
There is no need for wildcard, just use their domains.

Reply:
Changed the rule here.